tasks/install.yml

```diff
@@ -47,12 +47,30 @@
 
 - meta: flush_handlers
 
-- name: Install plugins
+- name: Set fluentd dirs ownership
```

Comment:
Why is this task in `install.yml` while [another task](https://github.com/cloudalchemy/ansible-fluentd/blob/f802b623315739517ae785e48b749fff3662fd4e/tasks/configure.yml#L2) which also sets file permissions is in `configure.yml`, couldn't it be unified?